docs/middleware/neutrino-middleware-eslint/README.md

```diff
@@ -37,6 +37,7 @@ const eslint = require('neutrino-middleware-eslint');
 // Usage shows default values
 neutrino.use(eslint, {
   test: /\.(js|jsx)$/,
+  extensions: ['js', 'jsx'],
```

Comment:
Should these be part of the eslint options?

Reply:
The option will be ignored when used by eslint-loader, but yeah it probably belongs in `eslint` anyway. I'll update the PR and add a comment.